wgsl/index.bs

```diff
@@ -824,6 +826,81 @@ See [[#arithmetic-expr]].
   </xmp>
 </div>
 
+### Atomic Types ### {#atomic-types}
```

Comment:
It strikes me that #1674 should be probably disallow atomics by value as a function parameter and return.

Reply:
Yes, should become "atomic-free plain types" like just about everything else.  But we can do that after both land.